Description: Fix crash on close (due to newer version of wxsqlite3)
Author: James Cowgill <james410@cowgill.org.uk>
Forwarded: https://github.com/eranif/codelite/pull/540
Applied-Upstream: https://github.com/eranif/codelite/commit/8238a26fb7ee5c2daacd690da20b8fb58cef6995
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CodeLite/tags_storage_sqlite3.h
+++ b/CodeLite/tags_storage_sqlite3.h
@@ -115,7 +115,9 @@ public:
     }
 
     void Close() {
-        wxSQLite3Database::Close();
+        if (IsOpen())
+            wxSQLite3Database::Close();
+
         m_statements.clear();
     }
 
